### ISSUE TITLE
VM does not use same nameservers as host

### ISSUE THREAD

**Issue by nvllsvm** [abiosoft/colima#4]
The VM does not use the same nameservers. This is necessary to pull images from an internal-only Docker repository. 

I fixed this by copying the contents of the host's /etc/resolv.conf into the VM's /etc/resolv.conf. 

Default /etc/resolv.conf
```
nameserver 127.0.0.53
options edns0 trust-ad
search .
```

Modified /etc/resolv.conf (values modified for example)
```
search internal.stuff
nameserver 1.2.3.4
nameserver 5.6.7.8
```

**Comment by abiosoft** [comment]
Good to hear you found a workaround.

Would you expect this to be the default behaviour?

**Comment by nvllsvm** [comment]
Yes - my expectation is that the VM uses the exact same DNS servers as the host.

**Comment by abiosoft** [comment]
The fix is imminent, the next version of Lima will support DNS override.
I am still leaning towards making it an explicit configuration, rather than making it the default.

**Comment by abiosoft** [comment]
This has been fixed upstream. i.e. DNS overrides can now be configured.
However, because Colima is currently being rewritten in Go, I would prefer to defer the feature until the rewrite is done.

**Comment by abiosoft** [comment]
this has been fixed in the Go rewrite and will be released as part of v0.2.0
